### ISSUE TITLE
generator model keys doesn't match

### ISSUE THREAD

**Issue by yeong95** [genforce/idinvert_pytorch#32]
I trained stylegan generator from genforce (https://github.com/genforce/genforce/tree/2ad04974cfaeba20b93c806531f987f06cc5c328)

But when loading weights from trained model the keys doesn't match each other.

How do i fix this?

![image](https://user-images.githubusercontent.com/52872215/184612068-b366a708-935d-49c2-b28b-e362f0499160.png)

**Comment by zhujiapeng** [comment]
The stylegan generator of genforce is refactored. The whole training pipeline can be found  [here](https://github.com/genforce/idinvert), this pytorch repo only supports inference.

**Comment by yeong95** [comment]
Thank you for reply.  

And one more question: Is the inversion process in genforce repo the same as the inversion process in this repo?

**Comment by zhujiapeng** [comment]
No, the genforce supports training the encoder in this [repo](https://github.com/genforce/ghfeat).
